### ISSUE TITLE
Python 3.7.16 is not availiable for Windows

### ISSUE THREAD

**Issue by ryan-rozario** [actions/setup-python#616]
**Description:**
Python 3.7.16 is not available for Windows in https://github.com/actions/python-versions/blob/main/versions-manifest.json
We need to update this version in our pipelines as part of mandatory security related updates. Could you please try to add this release.

**Action version:**
On AzureDevOps
Task: UsePythonVersion@0

**Platform:**
- [ ] Ubuntu
- [ ] macOS
- [x] Windows

**Runner type:**
- [x] Hosted
- [ ] Self-hosted

**Tools version:**

Python version 3.7.16
Python version 3.7.15
Python version 3.7.14
Python version 3.7.13
Python version 3.7.12
Python version 3.7.11
Python version 3.7.10

**Repro steps:**  
Run the action trying to use Python 3.7.15 on a windows machine
or
Run the action trying to use Python 3.7.16 

**Expected behavior:**
Install Python 3.7.16

**Actual behavior:**
```
Failed to download Python from the Github Actions python registry (https://github.com/actions/python-versions). Error: Error: Could not find Python matching spec 3.7.16 (x64) in the python-versions registry. Beware that only systems listed in the Github Actions python versions manifest (https://github.com/actions/python-versions/blob/main/versions-manifest.json) are fit for downloading python on-flight. Also, proxy is not supported.
```

**Comment by MaksimZhukov** [comment]
Hello @ryan-rozario!
Unfortunately, there are no files available related to both Python 3.7.15 and 3.7.16 for Windows ([Python Releases for Windows](https://www.python.org/downloads/windows/)), so we can't publish it.

**Comment by ryan-rozario** [comment]
Ok. Thanks for checking. I did not know that python does not provide windows executables for security updates. I feel expecting people to build from source would not encourage people to use the security releases.
